ui/Screen/Settings.svelte

```diff
@@ -144,85 +144,47 @@
 
     <section>
       <header>
-        <h3>Feedback</h3>
-      </header>
-      <div class="section-item">
-        <div class="info">
-          <p class="typo-text-bold">Get in touch directly</p>
-        </div>
-        <div class="action">
-          <a
-            class="typo-link"
-            href="https://radicle.community/c/site-feedback/2">
-            radicle.community
-          </a>
-        </div>
-      </div>
-      <div class="section-item">
-        <div class="info">
-          <p class="typo-text-bold">Join the developer chat</p>
-        </div>
-        <div class="action">
-          <a class="typo-link" href="irc://freenode:1/radicle">
-            #radicle on freenode
-          </a>
-        </div>
-      </div>
-    </section>
-
-    <section>
-      <header>
-        <h3>Peer ID</h3>
+        <h3>Devices</h3>
       </header>
       <div class="section-item">
         <div class="info">
-          <p>Share your peerID with others so they can add you as a remote</p>
+          <!-- TODO(abbey): link to actual docs abt multi-device support -->
+          <p>
+            Share your Device ID with others to be added as a remote. <br /><a
+              style="color: var(--color-foreground-level-5);"
+              class="typo-link"
+              href="https://radicle.xyz/#upstream-faq">Learn more about managing
+              devices.</a>
+          </p>
         </div>
         <div class="action">
           <StyledCopyable value={session.identity.peerId} />
         </div>
       </div>
     </section>
 
-    <section>
-      <header>
-        <h3>Appearance</h3>
-      </header>
-      <div class="section-item">
-        <div class="info">
-          <p class="typo-text-bold">Theme</p>
-        </div>
-        <div class="action">
-          <SegmentedControl
-            active={$settings.appearance.theme}
-            options={themeOptions}
-            on:select={updateTheme} />
-        </div>
-      </div>
-    </section>
-
     <section>
       <header>
         <h3>Network</h3>
       </header>
       <div class="section-item-single">
         <div class="info">
           <p class="typo-text-bold">
-            Seeds help you see more projects and people on the network
+            Seeds help you find more projects and people on the network.
           </p>
           <!-- TODO(julien): link to actual docs abt seeds -->
           <p
             style="color: var(--color-foreground-level-6); margin-bottom: 24px;">
             Enter seed addresses that you’d like to subscribe to here. <a style="color: var(--color-foreground-level-5);" class="typo-link" href="https://radicle.xyz/#upstream-faq"> Learn
-              more about seeds </a>
+              more about seeds. </a>
```

Comment:
Had this discussion with @brandonhaslegs about whether to put a full stop after a call to action before. I think the decision was to leave it out. The other place where we have a CTA is in the `RemoteHelperHint`.

Reply:
Hm. If we don't put a `.` then we should probably put a `:` or something no? Nothing looks weird to me. But if you have a strong preference @brandonhaslegs, I don't mind.

Reply:
Just for context, these are all of the places where we have CTAs as they are in this branch currently:

<img width="50%" alt="Screenshot 2020-11-10 at 14 08 37" src="https://user-images.githubusercontent.com/158411/98678091-541cbe80-235e-11eb-819a-4456344ab2dd.png">
<img width="70%" alt="Screenshot 2020-11-10 at 14 08 39" src="https://user-images.githubusercontent.com/158411/98678094-54b55500-235e-11eb-9530-0e083852ff6c.png">
<img width="50%" alt="Screenshot 2020-11-10 at 14 08 48" src="https://user-images.githubusercontent.com/158411/98678095-54b55500-235e-11eb-88f5-440a1c77eb6f.png">
<img width="50%" alt="Screenshot 2020-11-10 at 14 08 56" src="https://user-images.githubusercontent.com/158411/98678096-554deb80-235e-11eb-9b00-ad7d846d2bb3.png">

I'm only concerned about consistency -- so all the CTAs look the same within the app, I don't really care whether we go with full stop, or not.

> Hm. If we don't put a . then we should probably put a : or something no?

Not sure I follow, could you elaborate?

Reply:
Cool, I see what you mean. Let's remove the `.` in all CTAs and use that consistently moving forward

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1206/commits/e316a945bcd43ccc5137f9878d64bd0d59f4e0d1